src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,29 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+
+    private String name;
+
+    private ArrayList<Manager> Managers = new ArrayList<>();
+
+    public CEO(String name){ this.name = name; }
+
+    public CEO addManager(Manager m) {
+        Managers.add(m);
+        return this;
```

Comment:
Wow, FluentApi :D? Tego jeszcze nie było. Fajne jest korzystanie z takich metod łańcuchowych, mimo, że tutaj nie jest to jakieś mega potrzebne. Na plus, zaskoczyłeś mnie :)